srcpkgs/octoxbps/patches/0001-Fix-detection-of-Void-Linux-in-getLinuxDistro.patch

```diff
@@ -0,0 +1,39 @@
+From 0d06083d98b831e5b05bfb044c93dfe52c3cd9d5 Mon Sep 17 00:00:00 2001
+From: chili-b <dexter.gaonshatford@gmail.com>
+Date: Sat, 19 Mar 2022 15:33:41 -0400
+Subject: [PATCH] Fix detection of Void Linux in getLinuxDistro
+
+Before this change, OctoXBPS detects Void Linux based on whether or not
+the string, `PRETTY_NAME="void"`, occurs in the file, `/etc/os-release`.
+As of March 15 2022, the contents of `/etc/os-release` have been changed
+and `PRETTY_NAME="Void Linux"` replaces the previous value.
+(see https://github.com/void-linux/void-packages/commit/1dfc401d1237120e8e936627e7e28d2441916e80)
+
+This change is important because OctoXBPS only checks for the existence
+of the local package db if it detects that it is running on either Void
+Linux or Trident Linux. If OctoXBPS does not detect that it is running
+on either distribution (which it doesn't without this change), it will
+prompt for elevated privileges in order to synchronize the local package
+database with the remote database using `xbps-install -Syy` every time
+it is started, regardless of whether or not there are remote changes
+that need to be synced.
+---
+ src/unixcommand.cpp | 2 +-
+ 1 file changed, 1 insertion(+), 1 deletion(-)
+
+diff --git a/src/unixcommand.cpp b/src/unixcommand.cpp
+index 8a88a2f..ea84cd8 100644
+--- a/src/unixcommand.cpp
++++ b/src/unixcommand.cpp
+@@ -709,7 +709,7 @@ LinuxDistro UnixCommand::getLinuxDistro()
+ 
+       QString contents = file.readAll();
+ 
+-      if (contents.contains("PRETTY_NAME=\"void\""))
++      if (contents.contains("PRETTY_NAME=\"Void Linux\""))
```

Comment:
```suggestion
+      if (contents.contains("ID=\"void\""))
```
`ID` or `ID_LIKE` would be the proper way to check this, as it should never change, unlike `PRETTY_NAME`. (see also [the os-release spec](https://www.freedesktop.org/software/systemd/man/os-release.html))

Reply:
Done